web-registry/src/pages/CreditClassDetails/tables/CreditClassDetails.TableTabs.config.tsx

```diff
@@ -0,0 +1,30 @@
+import { IconTabProps } from 'web-components/lib/components/tabs/IconTab';
+
+import { CreditBatches } from 'components/organisms';
+
+import { AdditionalInfo } from '../CreditClassDetails.AdditionalInfo';
+import { CreditClassDetailsTableTabsProps } from './CreditClassDetails.TableTabs.types';
+
+export const getCreditClassDetailsTabs = ({
+  creditClassMetadata,
+  creditTypeName,
+  onChainCreditClassId,
+}: CreditClassDetailsTableTabsProps): IconTabProps[] =>
+  [
+    {
+      label: 'Credit Issuance',
+      content: <CreditBatches creditClassId={onChainCreditClassId} />,
+      hidden: !onChainCreditClassId,
+    },
+    {
+      label: 'Additional Info',
+      content: (
+        <AdditionalInfo
+          classId={onChainCreditClassId}
+          creditTypeName={creditTypeName}
+          metadata={creditClassMetadata}
+        />
+      ),
+      hidden: !creditClassMetadata,
```

Comment:
since we show the credit class id there too (which isn't part of the metadata), maybe we should always show this tab as soon as there is an `onChainCreditClassId`?

Reply:
https://github.com/regen-network/regen-web/pull/1985/commits/8945f1c2c24944fdb65fe644fe832d4f5c2be1d2